frontend/app.vue

```diff
@@ -1,5 +1,6 @@
 <script setup lang="ts">
 import type { NavigationMenuItem } from '@nuxt/ui'
+import { useAuthentication } from '~/composables/useAuthentication'
```

Comment:
imports like this (mostly composables and components) should be automatic in nuxt3, so there is no need to have this line. if the IDE says it's required, there might be something wrong with the IDE setup or the cache in the .nuxt folder.